### DISCUSSION TITLE
[Error] GetCollection failed: Unable to complete request, Not Found, Response: Unable to find '.tina/__generated__/_graphql.json'

### DISCUSSION THREAD

**Discussion by robbyph** [tinacms/tinacms#3092] [Help] [answered]
`[Error] GetCollection failed: Unable to complete request, Not Found, Response: Unable to find '.tina/__generated__/_graphql.json' in store for project 570e080a-46dc-460f-9765-a6d1e9409e5a: 'robbyph/robby-portfolio', Ref: 'main'`

I have no idea why, but last night, seemingly out of nowhere, this error started coming up. I was editing queries around that time, but even upon reverting these changes, the server continues to fail. I also added a collaborator to the Tina Cloud project, so i would also have the email i use for vercel on the Tina Cloud account, but not sure if that would have had any effect. 

This is happening in production on vercel with tinacms.

Not sure exactly what's going on, i've tried refreshing web hooks in the tina cloud client a few times and it's not really done anything. 

I went through the three recommended steps to fix this on the [FAQ page](https://tina.io/docs/tina-cloud/faq/), however, this yielded no fruit

1 - .tina/__generated__/_graphql.json is included in my github repository. Not in gitignore, and it is indeed on there.
2 - the .tina folder is present in the github repository.
3 - not sure about this one. i don't have a TinaCMS component in my _app.tsx, i have a TinaProvider component, which according to the docs does not take any kind of apiURL argument.

![There's about 5 errors that pop up when i try and pull up the admin panel, here they are if that helps!](https://user-images.githubusercontent.com/11604312/184383599-ffe8d9f1-4066-450e-a50b-25071555284d.png)

I'm kinda lost. Really not sure how to fix this. I've been going through the tina-cloud-starter repo just trying to make some of my schema and stuff match that because man i'm lost.

Thanks for any help you can provide :)

**Comment by kldavis4** [comment, marked as answer]
@robbyph can you try resetting the repository cache: https://tina.io/docs/tina-cloud/dashboard/projects/#reset-repository-cache ?

**Reply by robbyph** [reply]
@kldavis4 

Hey Kelly, thank you for responding so promptly! I reset the repository cache and redeployed in vercel and it's working now

Thank you so much :)

**Reply by robbyph** [reply]
@kldavis4 

Hey, after doing that, i thought it was all fixed, but now it is giving me the following error.

`IndexFailedError: Attempting to index DB responded with status 'failed', MergeNotSupportedError: Merges with conflicts are not supported yet.`

Is it appropriate to continue this in this thread, or would it be better to start a new one?

**Reply by kldavis4** [reply]
There is a bug that I'm currently working on around this issue with a fix hopefully out early next week. Can you try the reset again and see if it works around the issue for now?

**Reply by robbyph** [reply]
@kldavis4 Weird enough, the reset didn't fix it, but i had renamed my enviornmental variable for ClientID from 'TINA_ID' to 'NEXT_PUBLIC_TINA_CLIENT_ID' and it freaked out even though i had changed the name within Vercel. But, i changed it back and it works now lol, so crisis averted!

**Reply by kldavis4** [reply]
ok, great :) and feel free to reach out on discord in our #help channel if you have further issues: https://discord.gg/BxK5RxJn
